pkg/losetup/losetup_linux.go

```diff
@@ -0,0 +1,107 @@
+package losetup
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+)
+
+const (
+	/*
```

Comment:
is it worth checking to see if these are now defined in some extant go package? They did not used to be but ...

Reply:
I could not find them, any pointers on where to look ?

Reply:
Actually you are right the SYS_IOCTL cost is already present in syscall I need to change that.